data/Tonga/Assembly/ep-popolo-v1.0.json

```diff
@@ -904,17 +995,12 @@
       ]
     },
     {
-      "area_id": "area/vava'u_no._16",
```

Comment:
I've rescued this information by putting it into Wikidata, but the `id` has changed slightly, removing the `no._` part. I _think_ that's OK, but thought I'd explicitly flag it up.